org.freeplane.App.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.freeplane.App
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
```

Comment:
Is network access required?

Reply:
Considering, for example, you could embed URIs into the mindmaps I guess it does.